ait/core/api.py

```diff
@@ -91,7 +91,8 @@ def __str__(self):
 
     @property
     def msg(self):
-        s = 'FalseWaitError: "False" boolean passed as argument to wait. Ensure wait condition args are surounded by lambda or " "'
+        s = f'FalseWaitError: "False" boolean passed as argument to wait. Ensure wait ' \
```

Comment:
Our linter now complains when using format-strings without variable-substibution taking place, so could you please remove the 'f's?